build/bin/sage-spkg

```diff
@@ -584,14 +586,21 @@ if [ \$? -ne 0 ]; then
     exit 1
 fi
 
-cd "\$SAGE_PKG_DIR"
+__EOF__
+    if [ -n "$fallback_script_dir" ]; then
+        cat >> "$tmpscript" <<__EOF__
+cd "\$SAGE_PKG_DIR" 2>/dev/null || cd "$fallback_script_dir"
+
+__EOF__
+    else
+        cat >> "$tmpscript" <<__EOF__
 if [ \$? -ne 0 ]; then
     echo >&2 "Error: could not cd to the package build directory \$SAGE_PKG_DIR"
     exit 1
```

Comment:
Missing 
```
cd "\$SAGE_PKG_DIR"
```
before line 597 ?

Reply:
no, it's in line 592

Reply:
Line 592 is in "if then" clause...

Reply:
oh, I have to look at this again with less tired eyes

Reply:
Take your time.

Reply:
Of course, you were right.